modules/security/src/main/java/org/eclipse/xpanse/modules/security/IdentityProviderManager.java

```diff
@@ -43,33 +40,8 @@ public void loadActiveIdentityProviderServices() {
                 applicationContext.getBeansOfType(IdentityProviderService.class)
                         .values().stream().toList();
         if (CollectionUtils.isEmpty(identityProviderServices)) {
-            log.info("Not found any identity provider service is active.");
-            activeIdentityProviderService = new IdentityProviderService() {
-                @Override
-                public IdentityProviderType getIdentityProviderType() {
-                    return null;
-                }
-
-                @Override
-                public CurrentUserInfo getCurrentUserInfo() {
-                    return null;
-                }
-
-                @Override
-                public BackendSystemStatus getIdentityProviderStatus() {
-                    return null;
-                }
-
-                @Override
-                public String getAuthorizeUrl() {
-                    return null;
-                }
-
-                @Override
-                public TokenResponse getAccessToken(String code) {
-                    return null;
-                }
-            };
+            log.error("Not found any identity provider service is active.");
```

Comment:
Log line can be -> No active identity providers found.

Reply:
OK